resources/views/oauth2clientcreated.blade.php

```diff
@@ -0,0 +1,45 @@
+@extends('layouts/app')
+
+@section('title')
+    OAuth2 Client Details | {{ config('app.name') }}
+@endsection
+
+@section('content')
+    @component('layouts/title')
+        OAuth2 Client Details
+    @endcomponent
+
+    @if(Session::has("client_id"))
+        <p>
+            Your OAuth2 client was successfully created, and the client credentials are shown below.<br />
+            <small>Nova Actions can't show modals with information afterwards, so instead we had to send
+                you and your secrets to the Twilight Zone.</small>
+        </p>
+
+
+
+        @if(Session::has("client_plain_secret"))
+            <div class="alert alert-warning">
+                Save the client secret now—it will not be viewable after this!
+            </div>
+        @endif
+
+        <p>
+            <strong>Client ID:</strong> {{ Session::get("client_id") }}
+        </p>
+
+        <p>
+            @if(Session::get("client_confidential"))
+                <strong>Client Secret:</strong> {{ Session::get("client_plain_secret") ?? "Unavailable" }}
+            @else
+                <strong>Client Secret:</strong> This is a public (PKCE-enabled) client, so it doesn't have a client
+                secret.
+            @endif
+        </p>
+    @else
+        This page's content has expired.
```

Comment:
```suggestion
        This page's content has expired. This might happen if you run the action quickly after loading the user's page due to a known technical limitation.
```